src/pages/home/home.html

```diff
@@ -1,6 +1,6 @@
 <ion-header>
   <ion-navbar>
-    <ion-title>Home</ion-title>
+    <ion-title translate>Home</ion-title>
```

Comment:
Attributes inside the Ionic elements don't work.

Reply:
Updated. Thanks!